### PR TITLE
Change homepage url to point to docs

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

This PR changes the homepage url to point to https://mattermost.com/marketplace/app-framework

This is to match with the changes made in https://github.com/mattermost/mattermost-marketplace/pull/189#issuecomment-843396639

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "id": "com.mattermost.apps",
     "name": "Apps",
     "description": "This plugin powers the Mattermost Apps Framework and is required for any Apps to run",
-    "homepage_url": "https://github.com/mattermost/mattermost-plugin-apps",
+    "homepage_url": "https://mattermost.com/marketplace/app-framework",
```
Comment (hanzei) [inline plugin.json:5]: Given that this plugin currently targets developers should we link to https://developers.mattermost.com/integrate/apps/ instead to make it easier to get started?

Comment (DHaussermann) [reply plugin.json:5]: 0/5 @hanzei yes this is aimed at developers. But we're showing this to everyone and Aaron's page has a broader context. 

Also the related PR must go to the marketplace very soon. I propose we can open a follow-up PR for further tweaks in the next day or two so the URLs stay consistent between the repo and the marketplace.

**Review by hanzei — APPROVED** [review]

**Review by catalintomai — APPROVED** [review]

File: plugin.json (hunk 1 above)
Comment (hanzei) [reply plugin.json:5]: Makes sense to me

**Pull request merged**